config/jobs/kubernetes/sig-cloud-provider/azure/generate.sh

```diff
@@ -402,6 +403,11 @@ periodics:
     repo: kubernetes
     base_ref: ${branch_name}
     path_alias: k8s.io/kubernetes
+  - org: kubernetes-sigs
```

Comment:
Interesting, so these tests have been relying upon in-tree cloud-provider-azure all htis time? How confident are we that this cutover will be forward-compat w/ the `e2e-test` target of azurefile-csi-driver?

Reply:
I don't see any reason the CSI driver `e2e-test` target would make any assumptions on in-tree vs. out-of-tree but I guess we'll find out when this merges

Reply:
I can also run this locally if we want to be sure but given that the test is already broken I don't think it's worth holding this PR back

Reply:
ACK